controllers/cron_rollout_helpers.go

```diff
@@ -156,13 +160,15 @@ func overwriteDisruptionSelectors(ctx context.Context, cl client.Client, disrupt
 // CreateDisruptionFromTemplate constructs a Disruption object based on the provided owner, disruptionSpec, and targetResource.
 // The function sets annotations, overwrites selectors, and associates the Disruption with its owner.
 // It returns the constructed Disruption or an error if any step fails.
-func CreateDisruptionFromTemplate(ctx context.Context, cl client.Client, scheme *runtime.Scheme, owner metav1.Object, targetResource *chaosv1beta1.TargetResourceSpec, disruptionSpec *chaosv1beta1.DisruptionSpec, scheduledTime time.Time) (*chaosv1beta1.Disruption, error) {
+func CreateDisruptionFromTemplate(ctx context.Context, cl client.Client, scheme *runtime.Scheme, owner metav1.Object, targetResource *chaosv1beta1.TargetResourceSpec, disruptionSpec *chaosv1beta1.DisruptionSpec, scheduledTime time.Time, log *zap.SugaredLogger) (*chaosv1beta1.Disruption, error) {
 	disruption := createBaseDisruption(owner, disruptionSpec)
 
 	ownerNameLabel := getOwnerNameLabel(owner)
 	disruption.Labels[ownerNameLabel] = owner.GetName()
 
-	setDisruptionAnnotations(disruption, owner, scheduledTime)
+	if err := setDisruptionAnnotations(disruption, owner, scheduledTime); err != nil {
+		log.Warnw("unable to set annotations for child disruption", "err", err, "disruptionName", disruption.Name)
```

Comment:
apologies if i'm disagreeing with someone's earlier review, but I think I'd prefer this be an Error? When this happens, it means we couldn't unmarshal the UserInfo from the parent resource. That's a problem we want to be alerted on + diagnose